workloads/test_format/test_format.sh

```diff
@@ -0,0 +1,29 @@
+#! /bin/bash
+# Initial sample workload for use with the testy framework.
+
+describe() {
+    echo "A test/format workload for use with the testy framework."
+}
+
+populate() {
+    echo "Populate is not implemented in this wokrload."
+}
+
+run() {
+    ${script_dir}/testy-metrics.sh workload_status 1
+    ${wt_build_dir}/test/format/format.sh -c ${workload_dir}/test_format/${config_file} -h ${database_dir} -j nproc
```

Comment:
You will need the value of `nproc`:
```suggestion
    ${wt_build_dir}/test/format/format.sh -c ${workload_dir}/test_format/${config_file} -h ${database_dir} -j $(nproc)
```

Reply:
[ee37c51](https://github.com/wiredtiger/testy/pull/82/commits/ee37c511cd9c92c553a7135fc7ac92e3ee8bab1e)